app/build.gradle

```diff
@@ -168,7 +168,7 @@ project.gradle.taskGraph.whenReady {
 }
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
```

Comment:
Changed CompileSdkVersion from 31 to 33. Fixes error as described in this [stack overflow issue](https://stackoverflow.com/questions/69943549/one-or-more-issues-found-when-checking-aar-metadata-values).